setup.py

```diff
@@ -29,7 +29,10 @@
     "six>=1.9.0",
 )
 
-extras = {"aiohttp": "aiohttp >= 3.6.2, < 4.0.0dev; python_version>='3.6'"}
+extras = {
+    "aiohttp": "aiohttp >= 3.6.2, < 4.0.0dev; python_version>='3.6'",
+    "pyopenssl": "pyopenssl",
```

Comment:
Please add a lower bound (`pyopenssl >= x.x.x`). See https://www.pyopenssl.org/en/stable/changelog.html 

Otherwise LGTM.

Reply:
Yep, now made it `pyopenssl>=20.0.0`. In the change log, they mentioned they fixed a bug about `OpenSSL.crypto.load_privatekey` function (which we uses for mtls) in 20.0.0, so this version would be a good lower bound.